### PR TITLE
Editorial wrapping.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- editorial wrapping of http messages

## Note

- not sure this is readable... :thinking: :thinking: :thinking:   please take a look at the rendering in TXT/HTML and have your say

### CODE REVIEW

**Comment by jdesrosiers** [comment]
Personally, I find this kind of forced wrapping to be significantly harder to read than just having a few lines that are a little long. I'd rather scroll a little. Besides, 70 characters is a ridiculously short and unnecessary line limit in this day and age. If there's no good reason it needs to be that short, I'd vote to leave it how it is.

**Comment by ioggstream** [comment]
@reschke sorry for the silly question: can artworks be >70 chars? Do you have some references? Thx++, R.

**Comment by reschke** [comment]
artwork width is unfortunately limited, even know where TXT is only one of many output formats.

**Comment by ioggstream** [comment]
Thanks @reschke. @jdesrosiers it seems we have to stick to the limit, but you can adjust the wrapping in some other way.

File: draft-ietf-httpapi-rest-api-mediatypes.md [hunk 1]
```diff
@@ -373,16 +373,25 @@ by a unique JSON Schema.
...
+
 GET /pet/1234 HTTP/1.1
-Accept: application/schema-instance+json; schema="/schemas/v2/pet"; q=0.2,
-        application/schema-instance+json; schema="/schemas/v1/pet"; q=0.1
+Host: foo.example
+Accept: \
+  application/schema-instance+json; schema="/schemas/v2/pet"; \
+  q=0.2,  \
+  application/schema-instance+json; schema="/schemas/v1/pet"; \
+  q=0.1
```
Comment (jdesrosiers) [inline draft-ietf-httpapi-rest-api-mediatypes.md:384]: How about this. It comes in at exactly 70 chars and reads fine.

```suggestion
Accept: \
  application/schema-instance+json; schema="/schemas/v2/pet"; q=0.2, \
  application/schema-instance+json; schema="/schemas/v1/pet"; q=0.1
```

File: draft-ietf-httpapi-rest-api-mediatypes.md [hunk 2]
```diff
@@ -397,16 +406,26 @@ of JSON Schema and the server replies with the latest one.
 Request:
 
 ~~~ http-message
+NOTE: '\' line wrapping per RFC 8792
```
Comment (jdesrosiers) [inline draft-ietf-httpapi-rest-api-mediatypes.md:409]: Should this be a reference?

```suggestion
NOTE: '\' line wrapping per {{!rfc8792}}
```

File: draft-ietf-httpapi-rest-api-mediatypes.md [hunk 3]
```diff
@@ -397,16 +406,26 @@ of JSON Schema and the server replies with the latest one.
...
+
 GET /schemas/v2/pet HTTP/1.1
-Accept: application/schema+json; schema="https://json-schema.org/draft/2020-12/schema",
-        application/schema+json; schema="http://json-schema.org/draft-07/schema#"
+Host: foo.example
+Accept: \
+  application/schema+json; \
+  schema="https://json-schema.org/draft/2020-12/schema",\
+  application/schema+json; \
+  schema="http://json-schema.org/draft-07/schema#"
```
Comment (jdesrosiers) [inline draft-ietf-httpapi-rest-api-mediatypes.md:417]: How about this. A little more indentation could make it a little more readable.

```suggestion
Accept: application/schema+json; \ 
            schema="https://json-schema.org/draft/2020-12/schema", \
        application/schema+json; \
            schema="http://json-schema.org/draft-07/schema#"
```

File: draft-ietf-httpapi-rest-api-mediatypes.md [hunk 4]
```diff
@@ -397,16 +406,26 @@ of JSON Schema and the server replies with the latest one.
...
 ~~~
 
 Response:
 
 ~~~ http-message
-HTTP/1.1 200 OK
-Content-Type: application/schema+json; schema="https://json-schema.org/draft/2020-12/schema"
+NOTE: '\' line wrapping per RFC 8792
+
+HTTP/1.1 200 Ok
```
Comment (jdesrosiers) [inline draft-ietf-httpapi-rest-api-mediatypes.md:425]: Should be a capital "k".
```suggestion
HTTP/1.1 200 OK
```

File: draft-ietf-httpapi-rest-api-mediatypes.md [hunk 5]
```diff
@@ -397,16 +406,26 @@ of JSON Schema and the server replies with the latest one.
...
+Content-Type: \
+  application/schema+json; \
+  schema="https://json-schema.org/draft/2020-12/schema"
```
Comment (jdesrosiers) [inline draft-ietf-httpapi-rest-api-mediatypes.md:428]:
```suggestion
Content-Type: \
  application/schema+json; \
      schema="https://json-schema.org/draft/2020-12/schema"
```

File: draft-ietf-httpapi-rest-api-mediatypes.md (hunk 2 above)
Comment (ioggstream) [reply draft-ietf-httpapi-rest-api-mediatypes.md:409]: Artworks are heredocs :)

**Pull request merged**